skimage/data/tests/test_data.py

```diff
@@ -68,6 +68,14 @@ def test_coffee():
     data.coffee()
 
 
+def test_eagle():
+    """ Test that "eagle" image can be loaded. """
+    path = fetch('data/eagle.png')
```

Comment:
wouldn't `eagle = data.eagle()` work here?

Reply:
In this case, the test `test_cells_3d` must be changed as well...

Reply:
However, my guess was to avoid to use the `_load` function to test more directly the image.

Reply:
Yeah, I tried to be consistent with `test_cells_3d`. I could change both if you'd like @emmanuelle.

Reply:
no pb, it's fine for the test.

Reply:
the `test_cells_3d` test is a bit different because there is no corresponding function for the dataset in `skimage.data`. Here more code would be tested by calling `data.eagle()`. But it's not very important. the PR could also be merged as it.

Reply:
I can change only this one, then :slightly_smiling_face: